enhancements/validators.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Standard definition for the Validators (linters)
+authors:
+- "@camilamacedo86"
+  reviewers:
+- TBD
+  approvers:
+- TBD
+  creation-date: 2021-12-02
+  last-updated: 2021-12-02
+  status:implemented
+  see-also:
+- https://github.com/operator-framework/enhancements/pull/98
+---
+
+# Standard definition for external validators
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implemented`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+The Validators are static checks (linters) that can scan the manifests and provide
+with low-cost valuable results to ensure the quality of the distributions which will be distributed via OLM,
+to ensure that they respect some specific criteria for vendors and/or catalogs.
+
+For example, [Openshit](https://docs.openshift.com/container-platform/4.6/operators/admin/olm-managing-custom-catalogs.html)
+catalog is composed for more than one image. To publish via each image is required to respect some specific criteria
+such as to be part of the vendor catalog.
+
```

Comment:
I am not sure to follow. If I understand correctly I would rewrite and say something like:
Catalogs are often composed or more than one operator. Some criteria need to be met for publishing new or additional operators that may be specific to the vendor catalog.

Reply:
The OCP catalog is really composed of more than one image. We have the redhat/certified/community/marketplace index catalog to create the OCP catalog.

Reply:
I see but I would not call that "OCP catalog", possibly OpenShift catalogs but rather as [here](https://docs.openshift.com/container-platform/4.9/operators/understanding/olm-rh-catalogs.html) Red Hat-provided Operator catalogs

Reply:
I would avoid using the word `Catalog` here to mean a set of catalogs. In OLM nomenclature, `Catalog` is synonymous with a single source of content. The metaphor we generally try to follow is that a `Catalog` is equivalent to a package manager's `repository`. Similar to `dnf` or `yum` repositories of rpms.

So, in that context, I would rephrase this to say something like `Openshift's set of OLM content is composed of multiple Catalog Sources`